sogs/static/view_room.js

```diff
@@ -10,6 +10,19 @@ const makebuffer = (raw) => {
         realLength--;
     return b.subarray(0, realLength);
 };
+
+function formatBytes(bytes, decimals = 2) {
+    if (bytes === 0) return '0 Bytes';
+
+    const k = 1024;
+    const dm = decimals < 0 ? 0 : decimals;
+    const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
```

Comment:
I like your optimism

Reply:
hahah i thought might as well leave them in - maybe some day will be used somewhere